cpp/src/io/parquet/reader_impl.hpp

```diff
@@ -250,6 +247,39 @@ class reader::impl {
    */
   void decode_page_data(size_t skip_rows, size_t num_rows);
 
+  /*
+   *
+   *
+    Functions related to computing chunks and passes (reader_impl_chunking.cu)
+   *
+   *
+   */
```

Comment:
not a big fan of these headings in the code. IMO the names already suggest what the functions are related to.